app/backend/requirements.in

```diff
@@ -0,0 +1,15 @@
+azure-identity
+quart
+langchain
+openai[datalib]
+tiktoken
+azure-search-documents==11.4.0b6
```

Comment:
11.5.0-beta.4 is the latest
https://www.nuget.org/packages/Azure.Search.Documents/11.5.0-beta.4

Reply:
but if this causes some breaks we shouldn't pull it in

Reply:
For Python, we seem to have the latest? https://pypi.org/project/azure-search-documents/#history

Reply:
(This is the version we had before, by the way. I had to pin it as it's considered a pre-release version, but we need it for HnswParameters)

Reply:
ah, looking at the wrong feed. Yes, you are right!